### ISSUE TITLE
phi0914

### ISSUE THREAD

**Issue by PonteIneptique** [PerseusDL/canonical-latinLit#94]
- phi0914.phi001.perseus-eng1 has books in Latin -> Book 35 to 42 ( https://github.com/PerseusDL/canonical-latinLit/blob/master/data/phi0914/phi00135/phi0914.phi00135.perseus-eng1.xml )

CC @balmas @lcerrato

**Comment by balmas** [comment]
Unfortunately it looks to me like Livy is a complete mess and I'm not sure I want to merge the PR #96 until we can straighten it out.  We might have to go back to the original Perseus 4 source and start over.

It looks like there might have been various editions merged during the migration because the URNs for these in the classics.xml are completely screwed up and duplicated and cross editions.

**Comment by PonteIneptique** [comment]
I do not think we need to do over, as for what I have seen, texts are pretty consistent from an edition to the other (way they are introduced for example).

I would love to be able to add the missing stuff, I just do not know where to get it...

As far as PR goes, this PR is just as "wrong" as the original data (and by that I mean we miss some, data themselves are okay). I understand the principle, not sure I agree with the decision :')

**Comment by PonteIneptique** [comment]
I might actually be able to do it given the zip from Perseus. Is there any newer/more recent version of it ?

**Comment by balmas** [comment]
The P4 source files are here https://www.dropbox.com/s/y6bkfy7fjrj6q6e/Livy.tgz?dl=0
There are 2 separate directories: Classics and sdl -- in theory the data in sdl supercedes what's in Classics, but what is currently live in Perseus 4 is a mixture of data from sdl and Classics. 

The attached file contains the data from the Perseus 4 catalog file that drove the migration -- the format is Perseus 4 document id, cts urn, source text path

(the source text paths are slightly longer than in the P4 tar file linked here but I think you can figure it out --- I just removed the Livy and Latin/Livy from the path when zipping up for you.)

[livy.txt](https://github.com/PerseusDL/canonical-latinLit/files/64034/livy.txt)

**Comment by lcerrato** [comment]
see also #100
